src/components/Editor/Tab.js

```diff
@@ -173,6 +179,7 @@ class Tab extends PureComponent<Props> {
         onClick={() => selectSource(sourceId)}
         onContextMenu={e => this.onTabContextMenu(e, sourceId)}
         title={getFileURL(src)}
+        style={{ width: tabWidth }}
```

Comment:
Love padding curly braces like this as it is really easy to read, but for the sake of consistency it seems we should remove the spaces.